docs/includes/generated_docs/schemas/beta.tpp.md

```diff
@@ -227,7 +227,17 @@ return ordered_addrs.last_for_patient()
 <p class="dimension-indicator"><code>many rows per patient</code></p>
 ## apcs
 
+Admitted Patient Care Spells (APCS) data is provided via the NHS Secondary Uses Service.
 
+This table gives core details of spells.
+
+Each row is an in-hospital spell: a period of continuous care within a single trust.
+
+Refer to the [OpenSAFELY documentation on the APCS data source][apcs_data_source_docs]
+and the [GitHub issue discussing more of the background context][apcs_context_issue].
+
+[apcs_data_source_docs]: https://docs.opensafely.org/data-sources/apc/
+[apcs_context_issue]: https://github.com/opensafely-core/cohort-extractor/issues/186
```

Comment:
question: Should these reference links be included for all APCS-related tables or just the main `apcs` table? That is, should we duplicate the information, or is including once sufficient?